perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTest.java

```diff
@@ -5415,6 +5416,29 @@ public void deleteAttribute() throws Exception {
 		}
 	}
 
+	@Test
+	public void deleteLoginNamespaceAttributeFreesLogins() throws Exception {
+		System.out.println(CLASS_NAME + "deleteLoginNamespaceAttributeFreesLogins");
+
+		String deleteNamespace = "toDelete";
+		BlockedLogin del = new BlockedLogin("login1", deleteNamespace);
+		perun.getUsersManagerBl().blockLogins(sess, List.of(del.getLogin()), deleteNamespace, null);
+		String otherNamespace = "other";
+		BlockedLogin other = new BlockedLogin("login2", "other");
+		perun.getUsersManagerBl().blockLogins(sess, List.of(other.getLogin()), otherNamespace, null);
+		System.out.println(perun.getUsersManagerBl().getAllBlockedLoginsInNamespaces(sess));
+
+		AttributeDefinition attrDef = new AttributeDefinition();
+		attrDef.setFriendlyName("login-namespace:deleteNamespace");
+		attrDef.setNamespace("deleteNamespace");
```

Comment:
```suggestion
		attrDef.setNamespace("urn:perun:user:attribute-def:def");
```
different kind of namespace hehe (this is also why the tests fail)